host/host.go

```diff
@@ -0,0 +1,69 @@
+// Package host provides facilities for interfacing with the host operating
+// system, logging activities performed.
+package host
+
+import (
+	"log"
+	"os/exec"
+	"strings"
+
+	"github.com/taskcluster/shell"
+)
+
+// Run is equivalent to exec.Command(name, arg...).Run() but with logging.
+func Run(name string, arg ...string) (err error) {
```

Comment:
Uses of this will throw away stdout/stderr and not log successful command output from either, is that what we want?

Reply:
Yes, the use case here is when you only care about the output if something goes wrong. If you are interested in the output from a successful command, there is the `CombinedOutput` method which is the same, but retains the output.